src/styles/components/toggle-button.less

```diff
@@ -0,0 +1,48 @@
+.toggle-button {
+  display: none;
+
+
+  + label {
+    cursor: pointer;
+    position: relative;
+    padding-right: @toggle-width + @base-spacing-unit / 2;
+
+    &:before {
+      position: absolute;
+      top: 0;
+      right: 0;
+      display: block;
+      width: @toggle-width;
+      height: @toggle-height;
+      border-radius: @toggle-height;
+      border: rgba(0, 0, 0, 0.2) @toggle-border-width solid;
```

Comment:
#nit Can we order these shorthand values as `{width} {style} {color}`? That's how we do it elsewhere, and I would like for it to be consistent, even though it doesn't matter.